platform/barefoot/sonic-platform-modules-ingrasys/debian/sonic-platform-ingrasys-s9180-32x.postrm

```diff
@@ -0,0 +1,42 @@
+# Automatically added by dh_systemd_start
```

Comment:
i think this is automatically added by debian, remove from the check-in.

Reply:
removed file

Reply:
same comment as above